### PR TITLE
Add black beans and rice recipe

### LINKED ISSUE #12: Black beans with Lime and Coriander rice
I'm making black beans and lime and coriander rice tonight. If people like it I'll put up the recipe.

It's good to make in big quantities, but the beans take about 3 hours, they just need checking every 20 mins though.

### PR DESCRIPTION
**Pull request by ZooeyMiller**
Added the black beans and rice recipe cooked this week
fix #12

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ You know what this is for.
 ## Breakfast
 
 ## Mains
+[Black beans w/ Lime & Coriander rice (and sides)](black-beans.md)
```
Comment (m4v15) [inline README.md]: Maybe put it in a recipes/mains/ folder, like mine?

File: black-beans.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
+# Black beans with lime and coriander rice
```
Comment (m4v15) [inline black-beans.md:1]: This seems more than just black beans and rice recipe?

**Review by m4v15 — CHANGES_REQUESTED** [review]
Add to folder, maybe change the title to include the peppers and onion stuff?

File: black-beans.md (hunk 2 above)
Comment (ZooeyMiller) [reply black-beans.md:1]: fix'd

**Comment by ZooeyMiller** [comment]
@m4v15 please re-review given the changes.

**Comment by m4v15** [comment]
@ZooeyMiller I will when there are no more conflicts.

**Comment by ZooeyMiller** [comment]
@m4v15 solved merge conflicts. Please review.

**Review by m4v15 — APPROVED** [review]

**Review by Suuuuha — APPROVED** [review]

**Review by matthewdking — APPROVED** [review]

**Review by finnhodgkin — APPROVED** [review]

**Pull request merged**